diff -ru invoke-0.9.0/invoke/cli.py invoke-0.9.0-unbundle/invoke/cli.py
--- invoke-0.9.0/invoke/cli.py	2014-08-27 01:31:42.000000000 +0100
+++ invoke-0.9.0-unbundle/invoke/cli.py	2014-11-29 13:06:28.017164896 +0100
@@ -3,7 +3,7 @@
 import sys
 import textwrap
 
-from .vendor import six
+import six
 
 from .context import Context
 from .loader import FilesystemLoader, DEFAULT_COLLECTION_NAME
diff -ru invoke-0.9.0/invoke/collection.py invoke-0.9.0-unbundle/invoke/collection.py
--- invoke-0.9.0/invoke/collection.py	2014-08-25 23:18:26.000000000 +0100
+++ invoke-0.9.0-unbundle/invoke/collection.py	2014-11-29 13:07:00.954377064 +0100
@@ -2,8 +2,8 @@
 from operator import add
 import types
 
-from .vendor import six
-from .vendor.lexicon import Lexicon
+import six
+from lexicon import Lexicon
 
 from .parser import Context as ParserContext
 from .tasks import Task
diff -ru invoke-0.9.0/invoke/executor.py invoke-0.9.0-unbundle/invoke/executor.py
--- invoke-0.9.0/invoke/executor.py	2014-08-26 20:26:47.000000000 +0100
+++ invoke-0.9.0-unbundle/invoke/executor.py	2014-11-29 13:07:11.594446085 +0100
@@ -4,7 +4,7 @@
 from .util import debug
 from .tasks import Call
 
-from .vendor import six
+import six
 
 
 class Executor(object):
diff -ru invoke-0.9.0/invoke/monkey.py invoke-0.9.0-unbundle/invoke/monkey.py
--- invoke-0.9.0/invoke/monkey.py	2014-05-06 23:40:09.000000000 +0100
+++ invoke-0.9.0-unbundle/invoke/monkey.py	2014-11-29 13:07:21.939522570 +0100
@@ -3,7 +3,7 @@
 import select, errno, os, sys
 from subprocess import Popen as OriginalPopen, mswindows, PIPE
 
-from .vendor import six
+import six
 
 
 def read_byte(file_no):
diff -ru invoke-0.9.0/invoke/parser/context.py invoke-0.9.0-unbundle/invoke/parser/context.py
--- invoke-0.9.0/invoke/parser/context.py	2014-08-25 21:50:22.000000000 +0100
+++ invoke-0.9.0-unbundle/invoke/parser/context.py	2014-11-29 13:08:36.572996799 +0100
@@ -1,4 +1,4 @@
-from ..vendor.lexicon import Lexicon
+from lexicon import Lexicon
 
 from .argument import Argument
 
diff -ru invoke-0.9.0/invoke/parser/parser.py invoke-0.9.0-unbundle/invoke/parser/parser.py
--- invoke-0.9.0/invoke/parser/parser.py	2014-08-27 00:57:47.000000000 +0100
+++ invoke-0.9.0-unbundle/invoke/parser/parser.py	2014-11-29 13:09:03.048167626 +0100
@@ -1,8 +1,8 @@
 import copy
 
-from ..vendor.lexicon import Lexicon
-from ..vendor.fluidity import StateMachine, state, transition
-from ..vendor import six
+from lexicon import Lexicon
+from fluidity import StateMachine, state, transition
+import six
 
 # Mostly for importing via invoke.parser.<x>
 from .context import Context
diff -ru invoke-0.9.0/invoke/runner.py invoke-0.9.0-unbundle/invoke/runner.py
--- invoke-0.9.0/invoke/runner.py	2014-08-26 19:29:09.000000000 +0100
+++ invoke-0.9.0-unbundle/invoke/runner.py	2014-11-29 13:07:59.372756276 +0100
@@ -72,7 +72,7 @@
             err = "You seem to be on Windows, which doesn't support ptys!"
             raise PlatformError(err)
         # Proceed as normal for POSIX/etc platforms, with a runtime import
-        from .vendor import pexpect
+        import pexpect
 
         out = []
         def out_filter(text):
diff -ru invoke-0.9.0/invoke/tasks.py invoke-0.9.0-unbundle/invoke/tasks.py
--- invoke-0.9.0/invoke/tasks.py	2014-08-26 20:27:37.000000000 +0100
+++ invoke-0.9.0-unbundle/invoke/tasks.py	2014-11-29 13:08:18.189877921 +0100
@@ -5,8 +5,8 @@
 import inspect
 import types
 
-from .vendor import six
-from .vendor.lexicon import Lexicon
+import six
+from lexicon import Lexicon
 
 from .context import Context
 from .parser import Argument
